src/datalink/bpf.rs

```diff
@@ -27,11 +26,12 @@ use util::NetworkInterface;
 
 // NOTE buffer must be word aligned.
 #[inline]
-pub fn datalink_channel(network_interface: &NetworkInterface,
-                        write_buffer_size: usize,
-                        read_buffer_size: usize,
-                        channel_type: DataLinkChannelType)
-    -> io::Result<(Box<EthernetDataLinkSender>, Box<EthernetDataLinkReceiver>)> {
+pub fn datalink_channel
+    (network_interface: &NetworkInterface,
```

Comment:
Yes it is. I obey rustfmt all the time. Actually my editor automatically runs it on every save, so this just ended up here. I could back it out but before doing that I would like to ask you to consider going full rustfmt on the entire project instead. It's so nice to remove the human aspect of code style and make it fully automatic. I don't love every line rustfmt gives me, but at least it will be consistent and I don't have to argue with anyone about it because I didn't make the decision :)